parquet-benchmarks/pom.xml

```diff
@@ -73,6 +79,11 @@
 
   <build>
     <plugins>
+      <!-- This module disables semver checks because it is not a public API.
```

Comment:
The purpose of adding this plugin and commenting out at the same time is not clear to me. Could you please elaborate?

Reply:
The plugin is configured for all projects to avoid copying, but this one is internal and shouldn't be checked.